app/src/main/java/com/github/se/cyrcle/ui/review/AllReviewsScreen.kt

```diff
@@ -86,6 +100,21 @@ fun AllReviewsScreen(
                                           .testTag("ExpandedReviewBox$index")
                                           .clickable { setSelectedCardIndex(-1) }) {
                                     Column {
+                                      userViewModel.getUserById(curReview.owner)
+                                      Text(
+                                          text =
+                                              "Owner: ${userViewModel.currentUser.value?.username!!}",
```

Comment:
Can you really pretend "!!" ? Who knows if the user deletes his account... then it could become null.
Instead I think you could use "?: [DELETED_ACCOUNT]" or something like that.

Reply:
(Assuming the only reason a username is null is if the user was deleted)